app/src/main/java/org/simple/clinic/widgets/PatientSearchResultItemView.kt

```diff
@@ -93,6 +107,48 @@ class PatientSearchResultItemView(
     renderPatientPhoneNumber(searchQuery, model)
     renderVisited(model.lastSeen)
     renderLastSeen(model.lastSeen, currentFacilityId)
+    renderIdentifier(model.identifier, searchQuery)
+  }
+
+  private fun getId(identifier: Identifier, searchQuery: String): Id {
+    val id = when (identifier.type) {
+      BpPassport -> BpPassport.shortCode(identifier)
+      BangladeshNationalId -> identifier.value
+      EthiopiaMedicalRecordNumber -> identifier.value
+      IndiaNationalHealthId -> identifier.value
+      is Unknown -> throw IllegalArgumentException("Unknown identifier: $identifier")
```

Comment:
Won't throwing an error here cause a crash in older versions of the app if they don't know a particular type of ID which new app versions are creating?

Reply:
Good point. Should we just show the actual value of the `Unknown` type? We use that for search as well. So I think that makes sense.

Reply:
Yup, that should be the approach here.